test/remote/gateways/remote_stripe_test.rb

```diff
@@ -622,15 +622,15 @@ def test_invalid_login
   # These "track data present" tests fail with invalid expiration dates. The
   # test track data probably needs to be updated.
   def test_card_present_purchase
-    @credit_card.track_data = '%B378282246310005^LONGSON/LONGBOB^2205101130504392?'
+    @credit_card.track_data = '%B378282246310005^LONGSON/LONGBOB^2705101130504392?'
```

Comment:
These tests were failing with an "invalid card expiry date" error, so I updated the year from `22` to `27`